sympy/polys/ring_series.py

```diff
@@ -1158,3 +1158,97 @@ def rs_compose_add(p1, p2):
     if dp:
         q = q*x**dp
     return q
+
+
+class RingSeriesBase(object):
+    pass
+
+class RingSeries(RingSeriesBase):
+    """Sparse Multivariate Ring Series"""
+
+    def new(self, *args, **kwargs):
+        return self.__class__(*args, **kwargs)
```

Comment:
If ``RingSeriesBase`` inherits from say ``Basic``. You will not need this function. You can do something like ``self.func(*args, **kwargs)``